src/database/migrations/MigrateData.ts

```diff
@@ -99,7 +99,7 @@ export class MigrateData1695740345977 implements MigrationInterface {
 		for (const oldCustomer of await queryRunner.query(`SELECT * FROM customers`)) {
 			// 1. Create customer row:
 			console.info(`Creating CustomerEntity with id ${oldCustomer.customerId}`);
-			const customerEntity = new CustomerEntity(uuidv4(), oldCustomer.address);
```

Comment:
Is this "address" the cheqd address? If yes then I agree about switching to emails since it's more unique, whereas user might have more than one address (one for mainnet, one for testnet)